portfolio/src/main/webapp/script.js

```diff
@@ -32,13 +32,19 @@ function addRandomImage() {
   imageContainer.appendChild(image_element);
 }
 
-function loadTasks() {
+function commentsSection(){
+    var amount = document.getElementById("max-comments").value;
+    loadTasks(amount);
+}
+
+async function loadTasks(amount) {
   fetch('/data').then(response => response.json()).then((tasks) => {
-    const taskListElement = document.getElementById('task-list');
-    tasks.forEach((task) => {
-      taskListElement.appendChild(createTaskElement(task));
-    })
-  });
+    taskListElement = document.getElementById('task-list');
+    taskListElement.innerHTML = '';
+    for (let x = 0; x<amount; x++) {
+      taskListElement.appendChild(createTaskElement(tasks[x]));
+    }
+    });
```

Comment:
line 47: indent with 2 spaces instead of 4 to align with bracket opening in line 41